tsconfig.json

```diff
@@ -10,10 +10,9 @@
     "useUnknownInCatchVariables": true,
     "incremental": true,
     "tsBuildInfoFile": "./tsconfig.tsbuildinfo",
-    "preserveSymlinks": true,
     "paths": {
-      "@grafana/schema/dist/esm/*": ["../packages/grafana-schema/src/*"]
-    }
+      "@grafana/schema/dist/esm/*": ["../packages/grafana-schema/src/*"],
```

Comment:
this is really picky, but i don't think trailing commas are valid json. i'm kinda surprised this doesn't throw an error _somewhere_, it would if this was `package.json`

Reply:
btw maybe i'm wrong, did some googling and [apparently trailing commas are valid in tsconfig](https://github.com/TypeStrong/tsconfig/issues/30) so maybe it's not pure json 🤷